storage/wdpost/wdpost_run_faults.go

```diff
@@ -150,8 +150,6 @@ func (s *WindowPoStScheduler) declareRecoveries(ctx context.Context, dlIdx uint6
 // are faulty, and reports the faults on chain via the `DeclareFaults` message
 // to our miner actor.
 //
-// NOTE: THIS CODE ISN'T INVOKED AFTER THE IGNITION UPGRADE
```

Comment:
This comment should still be there, not sure why it got removed in the backport.